### DISCUSSION TITLE
Pd-dnd-plugin for drag and drop.

### DISCUSSION THREAD

**Discussion by TheGuyWhoo** [plugdata-team/plugdata#83] [General]
Check this out:

https://github.com/megrimm/pd-dnd-plugin

Just tested it and it works on linux. Doesn't appear require any building, just download the tkdnd binaries for the os, place them in the tkdnd folder and put the pd-dnd-plugin folder in your externals folder and it works.

**Comment by TheGuyWhoo** [comment]
Works in pure data I mean, not plugdata.

**Comment by timothyschoen** [comment]
Cool, I'll try it! Not a very high priority at the moment while there are still built-in objects that don't work (network stuff, text define, a few ELSE objects), but I'll look into it eventually.
